.github/workflows/create_release.yml

```diff
@@ -20,7 +20,13 @@ jobs:
           git config --global user.name "imodeljs-admin[bot]"
         name: git config
       - run: npm ci
-      - run: npx run-s lint build build:dist
+      - run: npx run-s lint build build:backend:webpack
+      - uses: actions/github-script@v2

```

Comment:
Does this action actually use powershell?  The [docs for it](https://github.com/marketplace/actions/github-script) make it seem like it's supposed to be JS...

Reply:
You're right. I was looking at the create release step, didnt realize it was JS